rmgpy/kinetics/surface.pyx

```diff
@@ -119,15 +129,31 @@ cdef class StickingCoefficient(KineticsModel):
         def __set__(self, value):
             self._T0 = quantity.Temperature(value)
 
+    property coverage_dependence:
+        """The coverage dependence parameters."""
+        def __get__(self):
+            return self._coverage_dependence
+        def __set__(self, value):
+            self._coverage_dependence = {}
+            if value:
+                for species, parameters in value.items():
+                    for variable, x in parameters.items():
+                        if variable is 'E':
+                            parameters[variable] = quantity.Energy(x)
+                        else:
+                            parameters[variable] = quantity.Dimensionless(x)
+                    self._coverage_dependence[species] = parameters
+
     cpdef double get_sticking_coefficient(self, double T) except -1:
         """
         Return the sticking coefficient (dimensionless) at temperature `T` in K. 
         """
-        cdef double A, n, Ea, T0, stickingCoefficient
+        cdef double A, n, Ea, T0, coverage_dependence, stickingCoefficient
```

Comment:
not used?

Reply:
ah, these got readded when I rebased everything yesterday